travis/install_grpc.sh

```diff
@@ -51,5 +51,5 @@ fi
 
 # Install gRPC python libraries from PyPI.
 # Dependencies like protobuf python will be installed automatically.
-grpcio_ver=1.6.0
+grpcio_ver=1.7.0
```

Comment:
1.7.1 here too, but they may not have it yet?

Reply:
They don't have 1.7.1 yet.